style.css

```diff
@@ -418,9 +415,13 @@ article h1 {
   flex-direction: row;
   justify-content: center;
   gap: 20px;
+  border-top: 2px solid #172b4d;
+  margin-top: 300px;
+  margin-bottom: 50px;
+  padding-top: 20px;
 }
 
-@media screen and (max-width: 2000px) {
+@media screen and (max-width: 3000px) {
```

Comment:
- [ ] great job using the media queries 💯 instead of using `max-width: 3000px` you can use min-width to ensure that your break point cover every possible screen size starting from your chosen breakpoint , kindly fix it 🙏 
_i suggest using `min-width: 1080px` but you can choose any any value between 768 and 1440 (PC SIZE)_

Reply:
Great work on implementing the changes requested by the reviewer! 🎉

As for your current design, I would suggest using a min-width instead of a max-width. By using `min-width: 1080px;`, your layout will adjust to fit larger screens, ensuring that your design looks good on a wide range of devices, including desktops, laptops, tablets, and smartphones. This will make your design more responsive, so it will look good on any device. Using a `max-width` without a `min-width` may lead to design issues when viewed on larger screens. Without a min-width, the design will not adjust to fit bigger screens, and will instead be cut off or have elements overlapping on bigger screens. Using a `min-width` will make the design more responsive and will ensure that it looks good on all devices. 💻 

Here's an example of how you can use min-width in CSS:
```
@media only screen and (min-width: 1080px) {
  /* your styles here */
}
```
You can also find more information about min-width and responsive design in the following resources:

[W3Schools - CSS Media Queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp)
[CSS Tricks - A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
Let me know if you have any other questions, and keep up the good work! 💻🎨